content/plugins/descriptions/jira.yaml

```diff
@@ -0,0 +1,97 @@
+# The server must be restarted to pick up changes in this file.
+
+# Required. Must be unique across all files in the directory. No whitespace. Use hyphens as
+# separators.
+name: jira
+# Required. Can have whitespace. Should be titleized.
+humanName: Jira
+heading: 'Backstage Jira Plugin'
+# Keep it short
+lead: 'See Jira summary in Backstage'
+
+seo:
+  # Don't forget to end with "| Roadie"
+  title: 'Backstage Jira Plugin | Roadie'
+  description: |
+    The Backstage Jira plugin integrates with Jira to show Jira
+    information inside Backstage where it can be associated with your project.
+
+logo:
+  # This must be a relative path. It should start without a slash or with ./
+  # The overall size of the file should be approximately 200 by 200.
+  # The image in the file should be approximately 100 by 100 pixels. It needs space around it.
+  # The image will be made greyscale by Gatsby Image Sharp.
+  fileSystemPath: './content/assets/logos/jira/jira_logo.png'
+  # The dimensions are required. They are the outer dimensions of the image file.
+  width: 200
+  height: 200
+
+coverImage:
+  # This must be a relative path. It should start without a slash or with ./
+  fileSystemPath: './content/assets/jira-plugin.png'
+  alt: 'A preview of Jira plugin including tasks summary, project information and Activity Stream.'
+
+# Instructions for someone who wants to use this plugin.
+# languages used here must be listed in the .babelrc
+gettingStarted:
+    # What will this step accomplish?
+  - intro: Install the plugin into Backstage.
+    language: bash
+    code: yarn add @roadiehq/backstage-plugin-jira
+  - intro: Add proxy config
+    language: yaml
```

Comment:
Got any idea why the YAML syntax highlighting doesn't work?

Reply:
@dtuite weird, from what I know syntax highlighter in Gastby use Prismjs that supports YAML files by default so it should work correctly..